totp-app/TOTP App.xcodeproj/project.pbxproj

```diff
@@ -303,13 +307,13 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = K8Y9P894R8;
+				DEVELOPMENT_TEAM = "";
```

Comment:
I see that DEVELOPMENT_TEAM was set to a non-empty string in browser sign in project, but changed from non-empty to empty string here. Why is that the case?

Reply:
D'oh, thanks, I forgot to reset the test values I had locally.